colearn_grpc/mli_factory_interface.py

```diff
@@ -25,6 +25,13 @@ class MliFactory(abc.ABC):
     """
     Interface a class must implement to be used as a factory by the GRPC Server
     """
+    _version = "0.2.6"
```

Comment:
We'll have to remember to update this... Looks like it could be imported from the setup.py file using pkgtools:
https://stackoverflow.com/questions/17583443/what-is-the-correct-way-to-share-package-version-with-setup-py-and-the-package

Reply:
Sounds good 👍